LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
OK, I find that interesting - why @jbarciauskas ? XBlock API is Apache to encourage xblocks of all license types, but I don't quite understand why an xblock we author shouldn't be agpl